platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/main/java/module-info.java

```diff
@@ -6,8 +6,10 @@
     requires transitive com.swirlds.platform.core;
     requires com.swirlds.config.api.test.fixtures;
     requires com.swirlds.config.api;
+    requires com.swirlds.logging;
```

Comment:
Were the modifications in this file intentional?

Reply:
Yes, unfortunately. The CI job that tells you how to change the `module-info.java` files required me to make this change.